charts/kuberpult/values.yaml

```
@@ -242,6 +242,10 @@ frontend:
     timeout: 2m
   # The maximum message size in megabytes the client can receive.
   grpcMaxRecvMsgSize: 4
+  # The enableRevisions option is EXPERIMENTAL and may be removed or changed without a breaking change.
+  # When enabled, the /release endpoint to accept a revision (i.e. 1.0.0, 1.1.3...). Releases are ordered
+  # by revision and not by version number.
+  enableRevisions: true
```

if it's experimental, it must not be enabled by default
```suggestion
  enableRevisions: false
```